backend/director/tools/fal_video.py

```
@@ -90,3 +104,48 @@ def text_to_video(self, *args, **kwargs):
+
+                        image_url_result = res["images"][0]["url"]
+
+                        async with session.get(image_url_result) as image_response:
+                            with open(save_at, "wb") as f:
+                                f.write(await image_response.read())
+                        break
+                    else:
+                        raise ValueError(f"Unknown status for FAL request: {status_json}")
+                    
+        except Exception as e:
+            raise Exception(f"Error generating image: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)